var/spack/repos/builtin/packages/hdf5-vol-log/package.py

```diff
@@ -13,9 +13,11 @@ class Hdf5VolLog(AutotoolsPackage):
     homepage = 'https://github.com/DataLib-ECP/vol-log-based'
     url      = 'https://github.com/DataLib-ECP/vol-log-based'
     git = 'https://github.com/DataLib-ECP/vol-log-based.git'
-    maintainers = ['hyoklee']
+    maintainers = ['hyoklee', 'lrknox']
 
-    version('master', commit='28b854e50c53166010d97eccdc23f7f3ef6a5b03')
+    version('master-1.1', branch='master')
+
+    version('1.1.0', tag='logvol.1.1.0')
```

Comment:
Can you add the SHA of the commit to protect this against changes (malicious or unintentional)?

Reply:
I checked the github url for DataLib-ECP/vol-log-based: https://github.com/DataLib-ECP/vol-log-based/releases/tag/logvol.1.1.0  where the 1.1.0 release source is posted.  There is a tag and a commit hash posted, but I don't find a SHA.  I can generate one from the file that spack install --keep-stage hdf5-vol-log has downloaded, but I suppose the SHA needs to be posted on the release page?

Reply:
Sorry, by SHA I mean the git commit SHA -- the `commit=` argument.

Reply:
Yes, I misunderstood.  I can do that.